### PR TITLE
Explore 2.0

### PR DESCRIPTION
**Pull request by leem42**
Can view a demo of it here https://leem42.github.io/Synapse-React-Client/Playground/QueryWrapperPlotNavDemo

### CODE REVIEW

File: src/__tests__/lib/containers/widgets/facet-nav/FacetNav.test.tsx [hunk 1]
```diff
@@ -120,17 +120,6 @@ describe('facets display hide/show', () => {
       true,
     )
   })
-  it("should only show specified facets if 'facetsToPlot' are set", async () => {
```
Comment (conundrumgirl) [inline src/__tests__/lib/containers/widgets/facet-nav/FacetNav.test.tsx:123]: why did you remove it?

File: src/lib/containers/widgets/facet-nav/FacetNav.tsx [hunk 2]
```diff
@@ -40,33 +40,31 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
...
   const request = getLastQueryRequest!()
-  const getFacets = (data: QueryResultBundle | undefined): FacetColumnResult[] =>  {
+  const getFacets = (
+    data: QueryResultBundle | undefined,
+  ): FacetColumnResult[] => {
     const result = data?.facets?.filter(
-      item =>
-        item.facetType === 'enumeration' &&
-        (!facetsToPlot?.length || facetsToPlot.indexOf(item.columnName) > -1)
+      item => item.facetType === 'enumeration',
```
Comment (conundrumgirl) [inline src/lib/containers/widgets/facet-nav/FacetNav.tsx:56]: e we are not supporting the ability to select the facets to display in vis?

File: src/lib/containers/widgets/query-filter/QueryFilter.tsx [hunk 3]
```diff
@@ -55,7 +54,7 @@ const patchRequestFacets = (
   changedFacet: FacetColumnRequest,
   lastRequest?: QueryBundleRequest,
 ): FacetColumnRequest[] => {
-  const selections = lastRequest ? lastRequest.query.selectedFacets || [] : []
+  const selections = lastRequest?.query?.selectedFacets ?? []
```
Comment (conundrumgirl) [inline src/lib/containers/widgets/query-filter/QueryFilter.tsx:57]: that's a lot of question marks on one line:)

File: src/lib/style/components/_query-wrapper-plot-nav.scss [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.QueryWrapperPlotNav {
+  .QueryWrapperPlotNav__title {
```
Comment (conundrumgirl) [inline src/lib/style/components/_query-wrapper-plot-nav.scss:2]: you can just do 
.QueryWrapperPlotNav {
&__title {...}
&__querycount {...}
&__actions {}
}
 same thing functionally

File: src/lib/containers/widgets/facet-nav/FacetNav.tsx (hunk 2 above)
Comment (leem42) [reply src/lib/containers/widgets/facet-nav/FacetNav.tsx]: Whoops, let me add that back, took it out for an unrelated reason but forgot to add it back.
